samples/fsharp/getting-started/DeepLearning_TensorFlowEstimator/ImageClassification.Train/assets/outputs/README.txt

```diff
@@ -0,0 +1 @@
+output gets saved to this directory
```

Comment:
@CESARDELATORRE  we don't have this file in C# sample. I think the name **outputs** itself describes the output files. I feel no need to add this file. Cesar Please add your inputs.

Reply:
I put it so the folder gets created even if there's no model .ZIP file in there.. But I agree that this might not be needed. I'll merge it in any case. Thanks! 👍

Reply:
Was only to force the directory to be created in the repo, following the example of other projects (for example outputs directory of customer segmentation). Though the usual message is typically
```
This folder is empty until you run the training console app and create/train and save the models as .ZIP files.
Those model .ZIP files should appear in this folder.
```
Alternatively, could add a .gitignore to force the directory or add a `CreateDirectory` to the code.